sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -1329,8 +1330,12 @@ static class SchemaPojo {
   }
 
   @Test
-  public void testSchemaWriteLoads() throws Exception {
-    assumeTrue(!useStreaming);
+  public void testBatchSchemaWriteLoads() throws Exception {
+    // This test is actually for batch!
+    // We just invert useStreaming because we need it as true to test
+    // STORAGE_API_AT_LEAST_ONCE,
+    assumeTrue(useStreaming);
```

Comment:
It looks like you don't invert useStreaming, is that intentional?

Reply:
I meant that we're meant to use `!useStreaming` for batch tests, but used `useStreaming` instead. Will update comments to clarify